addons/api/addon/generated/models/managed-groups.js

```diff
@@ -0,0 +1,54 @@
+import BaseModel from '../../models/base';
+import { attr } from '@ember-data/model';
+
+/**
+ * A managed group is a resource that represents a collection of accounts.
+ */
+export default class GeneratedManagedGroupModel extends BaseModel {
+  // =attributes
+
+  @attr('string', {
+    description: 'Optional name for identification purposes.',
+  })
+  name;
+
+  @attr('string', {
+    description: 'Optional user-set description for identification purposes.',
+  })
+  description;
+
+  @attr('date', {
```

Comment:
from where do we take these desc strings?

Reply:
From [the proto file](https://github.com/hashicorp/boundary/blob/main/internal/proto/local/controller/api/resources/managedgroups/v1/managed_group.proto#L27-L28) made by BE. Basically copy/paste their comment